react/features/rtcstats/middleware.js

```diff
@@ -150,7 +150,7 @@ MiddlewareRegistry.register(store => next => action => {
         break;
     }
     case ADD_FACE_EXPRESSION: {
-        if (canSendRtcstatsData(state)) {
+        if (canSendRtcstatsData(state) && faceLandmarks.enableRTCStats) {
```

Comment:
What if there is no `faceLandmarks` in config.js?

Reply:
I see now that I will raise an error. I'll check if faceLandmarks is defined.

Reply:
Fixed it.